LandSandBoat/server
# Fix Caederva Mire Logging Point IDs

**_I affirm:_**

## What does this pull request do?

The Caedarva Mire Logging table has incorrect IDs which causes uninteractable NPCs to appear in trees. This fixes the IDs to start at the Logging Point IDs nearby in the npc_list.sql.
![image](https://user-images.githubusercontent.com/109392529/185268687-360bea46-dcff-4245-aff6-b2a6a14a8b12.png)

## Steps to test these changes

1. Go to the coordinates listed for the logging point entries.
2. Verify that the trees contain logging points and not Nashmeira.
